ApollosProject/apollos-prototype
# Cleans up Project

## DESCRIPTION

### What does this PR do, or why is it needed?

We had a ton of "stale" configuration laying around, plus some misconfiguration caused by a bad release last time around that I missed. Below is a rough list of the changes I made.

- Removed all configuration related to `now`
- Removed extranious code around the `yarn start` commands related to `now`. 
- Removed unused variables in `.env` files.
- Incremented package versions that were stuck on 0.8.5
- Incremented lerna version so I can pass `--force-publish` (not going to pull the trigger on this one yet)


### What design trade-offs/decisions were made?

### How do I test this PR?

Run the `yarn start` commands. Make sure you can walk through your normal dev flow 🤷‍♂ 

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
